src/lib/commandline_cmds.ts

```diff
@@ -160,7 +160,8 @@ export function getCommandlineFns(cmdline_state: {
             // Save non-secret commandlines to the history.
             if (
                 !browser.extension.inIncognitoContext &&
-                !(func === "winopen" && args[0] === "-private")
+                !(func === "winopen" && args[0] === "-private") &&
```

Comment:
Maybe we should support zero to totally disable saving of history?

Reply:
I does not like the idea that do `State.getAsync('historycmdcountlimit')` everytime before save ex command;
but we already do this while save history.